flow/scenarios/__init__.py

```diff
@@ -25,20 +27,12 @@
 from flow.scenarios.netfile.scenario import NetFileScenario
 from flow.scenarios.loop_merge.scenario import TwoLoopsOneMergingScenario
 
-# base scenario class
-__all__ = ["Scenario"]
-
-# custom generators
-__all__ += [
-    "BayBridgeGenerator", "BayBridgeTollGenerator", "BottleneckGenerator",
-    "Figure8Generator", "SimpleGridGenerator", "HighwayGenerator",
-    "CircleGenerator", "MergeGenerator", "NetFileGenerator",
-    "TwoLoopOneMergingGenerator"
-]
-
-# custom scenarios
-__all__ += [
-    "BayBridgeScenario", "BayBridgeTollScenario", "BottleneckScenario",
-    "Figure8Scenario", "SimpleGridScenario", "HighwayScenario", "LoopScenario",
-    "MergeScenario", "NetFileScenario", "TwoLoopsOneMergingScenario"
+__all__ = [
```

Comment:
Why do this? This seems less clean.

Reply:
Docstring conventions don't like it when we do this:
```
WARNING: Error in file flow/scenarios/__init__.py: Could not evaluate contents of __all__. 
That means pydocstyle cannot decide which definitions are
public. Variable __all__ should be present at most once in
each file, in form
```

Reply:
kk, can't do much about that